src/System.Text.Primitives/System/Text/Parsing/Unsigned.cs

```diff
@@ -0,0 +1,301 @@
+﻿
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// NOTE: This file is generated via a T4 template. Please do not edit this file directly. Any changes should be made
+// in Unsigned.tt.
+
+namespace System.Text
+{
+    public static partial class PrimitiveParser
+    {
+        public static bool TryParseByte(ReadOnlySpan<byte> text, out byte value, out int bytesConsumed, EncodingData encoding = default(EncodingData), TextFormat format = default(TextFormat))
+        {
```

Comment:
Should we throw, or just return false?

Also, are we ever planning on supporting formats where the number of digits is specified? It makes sense to support these from a formatting perspective, but perhaps not from a parsing perspective since it seems like the only effect would be to invalidate some strings that don't match the format.